now.json

```diff
@@ -13,6 +13,11 @@
     },
     {
       "handle": "filesystem"
+    },
+    {
+      "src": "/([a-z]+)/.*",
+      "status": 404,
+      "dest": "/$1/404"
```

Comment:
This should do the trick for the ZEIT routing.

Reply:
In my migration branch I opted to always point to the 404 of the `/docs` homepage path ( `/docs/404` ) - that also catches   paths like  `/12`  without showing the ZEIT 404.  

If we think there's a use case for microsite-specific 404s the approaches can be combined, too - add another route behind yours that points all else to  `/docs/404`

Reply:
PS: obviously, here in the kit repo this is all not a problem.

Reply:
Yeah, if we have a "home" microsite we can do that as well.